src/formatly.ts

```diff
@@ -17,7 +17,10 @@ export interface FormatlyReportError {
 export interface FormatlyReportResult {
 	formatter: Formatter;
 	ran: true;
-	result: Result;
+	result: {
+		code: null | number;
+		signal: NodeJS.Signals | null;
+	};
```

Comment:
[Feature] Could you create a named type for this and export it, please? I've found that eventually someone will want to use any given exported object shapes on its own.  `FormatlyReportResult['result']` is a little clunky.

https://github.com/typescript-eslint/typescript-eslint/issues/7670 strikes again 🤘

Reply:
I exported it as `FormatlyReportChildProcessResult`. Not sure if that's the perfect name but let me know if you have another in mind.